src/validation.jl

```diff
@@ -35,4 +42,23 @@ function same_lngth_shifts(data::Dict)
     "OK"
 end
 
+function contains_all_or_none_penalties(data::Dict)
+    schedule_priority = get(data, "penalty_priorities", nothing)
+
+    if !isnothing(schedule_priority)
+        default_priority = JSON.parsefile("config/default.json")["penalties"]
+        sort!(schedule_priority)
+        sort!(default_priority)
+        if length(default_priority) != length(schedule_priority)
+            "wrong priorities length, received '$(length(schedule_priority))', excpected '$(length(default_priority))'"
+        elseif !isequal(default_priority, schedule_priority)
```

Comment:
Te sorty z wyżej można tutaj zrobić.

Reply:
No można. Tak ułożyłem bo po prostu jakby nie zachodził warunek to nie jest potrzebny sort.